### DISCUSSION TITLE
Import bookmarks from browser.

### DISCUSSION THREAD

**Discussion by Tiritibambix** [beromir/Servas#29] [Q&A] [answered]
Hello.

Is there any way I can import my bookmarks from my browser and sort them afterwards in servas ?

**Comment by beromir** [comment, marked as answer]
Hi,
unfortunately at the moment this is not possible. I know this is a very requested feature, but I am currently working on other functions first. The function is on my list and I will work on it eventually.

**Comment by Tiritibambix** [comment]
Hello. I'm thrilled to see that this feature is implemented. thank you and good job.

However, I see it doesn't support the Netscape HTML format. Is this something that will be supported in the future ?

**Reply by beromir** [reply]
Yes, I am working on it. The JSON format is currently mainly for backing up your data and migrating to another instance.
Will let you now when the HTML import is possible.
